app/models/concerns/patient_reportable.rb

```diff
@@ -10,20 +10,15 @@ module PatientReportable
     scope :excluding_dead, -> { where.not(status: :dead) }
 
     scope :ltfu_as_of, ->(time) do
-      last_bp_after = (time - LTFU_TIME).end_of_month
-      joins("LEFT OUTER JOIN latest_blood_pressures_per_patient_per_months
-             ON patients.id = latest_blood_pressures_per_patient_per_months.patient_id
-             AND #{sanitize_sql(["bp_recorded_at > ? AND bp_recorded_at < ?", last_bp_after, time])}")
-        .where("bp_recorded_at IS NULL AND patients.recorded_at <= ?", last_bp_after)
+      left_outer_joins(:patient_states)
+        .merge(Reports::PatientState.where(month_date: time.beginning_of_month, htn_care_state: "lost_to_follow_up"))
```

Comment:
`PatientState` has a `by_month_date` that can be used here.

Reply:
[suggestion] We could also move the `htn_care_state` clause to a scope to `PatientState`, which would abstract the actual implementation. Would look something like
```
Reports::PatientState.by_month_date(time.beginning_of_month).lost_to_follow_up
````